src/index.ts

```diff
@@ -38,9 +56,15 @@ export const setSiteId = (siteId: string): void => {
   fathom('set', 'siteId', siteId);
 };
 
-export const trackPageview = (): void => {
+export type PageViewOptions = {
+  url?: string;
+  referrer?: string;
+}
+
+export const trackPageview = (opts?: PageViewOptions): void => {
   let fathom = getFathom();
-  fathom('trackPageview');
+  if (opts) fathom('trackPageview', opts);
+  else fathom('trackPageview');
```

Comment:
I think this might be a typo.

```js
opts ? fathom('trackPageview', opts) : fathom('trackPageview');
```

Reply:
Updated this in eb1076f8f60b15585941a37e679e65f211c79f24 :)